src/wallet/direct_send.rs

```diff
@@ -147,31 +173,21 @@ impl Wallet {
             }
         };
 
-        output.push(txout);
+        tx.output.push(txout);
 
         // Only include change if remaining > dust
         if let SendAmount::Amount(amount) = send_amount {
```

Comment:
Dust limits only affect our change address creation. If below dust, don't create the change. That scenario is already covered. Only thing can be added here is a nice info message saying change was below dust, hence skipped.